modules/headway.yml

```diff
@@ -0,0 +1,14 @@
+name: headway
+description: Headway changelog widget for Nuxt
+repo: l-portet/nuxt-headway
+npm: nuxt-headway
+icon: https://i.ibb.co/mFw3hxv/headway.png
```

Comment:
Can you please add the icon into https://github.com/nuxt/modules/tree/main/static/icons ?

Reply:
Sure, should be updated now.